04.ls/wc.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require 'optparse'
+
+OPTION_FLAG_TO_STAT_KEY = {
+  l: :lines,
+  w: :words,
+  c: :bytes
+}.freeze
+
+def main
+  option_flag = parse_options
+  if !ARGV.empty?
+    process_files(option_flag)
+  elsif !$stdin.tty?
```

Comment:
```suggestion
  else
```

でもいいんじゃないかな、と思ったのですが、どうでしょう？

参考 https://bootcamp.fjord.jp/pages/write-code-write-test

Reply:
これ、対応されてないのでは？？

Reply:
こちら、いかがですか？